src/components/quickStarts.stories.mdx

```diff
@@ -0,0 +1,470 @@
+<Meta title="Quick Starts" />
+
+# Getting Started
+
+## Setup the Environment
+
+1. Add amazon-chime-sdk-component-library-react and the necessary peer dependencies to your project
+
+```
+npm install --save amazon-chime-sdk-component-library-react amazon-chime-sdk-js styled-components styled-system
```

Comment:
Whats the difference between this story and introduction.stories? We dont want any duplicates if possible - it will be too hard to maintain

Reply:
Yeah, I'm not sure to be honest, feels like some parts are duplicated. I need to confirm with Vidya about this when she went back. She wants me add the how to for meeting join, which has already been covered in introduction.stories.

Reply:
Maybe we could merge them later